consensus/src/block_graph.rs

```diff
@@ -856,7 +856,7 @@ pub struct BlockGraph {
     to_propagate: BlockHashMap<(Block, OperationHashSet, Vec<EndorsementId>)>,
     attack_attempts: Vec<BlockId>,
     new_final_blocks: BlockHashSet,
-    new_stale_blocks: BlockHashMap<Slot>,
+    new_stale_blocks: BlockHashMap<(PublicKey, Slot)>, // creator, slot
```

Comment:
It may be clearer to use a named struct

Reply:
since it's just 2 elements, I prefer not creating a structure just for that. The comment is quite clear :)

Reply:
An alternative could be defining a type for this context or use import .. as ..

Reply:
true, but let's leave it like this for now as the tuple is small and heavily explicit :)